examples/mnist_poison_detection.py

```diff
@@ -66,7 +66,7 @@ def main():
 
     model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['accuracy'])
 
-    classifier = KerasClassifier((min_, max_), model=model)
+    classifier = KerasClassifier(model=model)
```

Comment:
`clip_values` is now a keyword argument. We'd like to keep the clip_values in the example to demonstrate its use. How about this:
```suggestion
    classifier = KerasClassifier(model=model, clip_values=(min_, max_))
```

Reply:
fine